parse_pom_sequence.sh

```diff
@@ -0,0 +1,83 @@
+#!/bin/bash
+#
+# Common utilities used by tools for automating tagging and release
+#
+
+#We want any failures to be fatal
+# Depends: maven
+
+set -e
+
+if ! `which mvn 1>/dev/null 2>&1` ; then
+  echo "Maven dependency not located"
+  exit 2
+fi
+
+# overwrite versionfile to pom.xml
+VERSIONFILE=pom.xml
+if [ ! -f $VERSIONFILE ]; then
+  echo "Expected $VERSIONFILE not found"
+  exit 2
+fi
+
+
+# overwrite parse_cmake to target pom files
+# expect ex: 7.6.2-0-SNAPSHOT, 7.6.2-1 (gold release), etc
+# Major.Minor.Point- Sequence if gold, Maturity if rc
+function parse_cmake()
+{
+  HPCC_PROJECT=$(mvn help:evaluate -Dexpression=project.artifactId -q -DforceStdout)
+  HPCC_VERSION=$(mvn help:evaluate -Dexpression=project.version -q -DforceStdout)
+  HPCC_MAJOR=$(echo $HPCC_VERSION | awk 'BEGIN {FS="[.-]"}; {print $1};')
+  HPCC_MINOR=$(echo $HPCC_VERSION | awk 'BEGIN {FS="[.-]"}; {print $2};')
+  HPCC_POINT=$(echo $HPCC_VERSION | awk 'BEGIN {FS="[.-]"}; {print $3};')
+  HPCC_SEQUENCE=$(echo $HPCC_VERSION | awk 'BEGIN {FS="[.-]"}; {print $4};')
+  HPCC_MATURITY=$(echo $HPCC_VERSION | awk 'BEGIN {FS="[.-]"}; {print $5};')
```

Comment:
If any of these HPCC_XYZ vars end up as empty strings, we should log and stop the process

Reply:
Added in a catch which will error out if we're using the versioning system without a HPCC_SEQUENCE